### DISCUSSION TITLE
Jira Integration

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#2861] [Ideas]
Hi There!
It will amazing if Outline would have integration with jira board and tasks. Is this integration in the roadmap?

**Comment by ali-master** [comment]
Any news?

**Reply by ali-master** [reply]
We want to use the outline instead of confluence but there is no Jira integration.

**Reply by Limezy** [reply]
I had a very quick look and apparently it's not possible displaying a Jira board inside an iFrame.
Apart from very specific cases such as Mermaid that has been integrated into Outline's codebase, rendering through an iFrame seem to be the way to go for any Outline integrations. Because of that, I believe it's quite unlikely that this will happen anytime soon.

**Reply by ali-master** [reply]
We want to access to the jira issues in details. Is it possible?

**Comment by florian-obradovic** [comment]
I would also love to see this :)
Here an example: https://drupal-wiki.com/jira-integration/

**Comment by jrevillard** [comment]
Very interested into this too !

**Comment by Nepo26** [comment]
This should be possible. This is an example of an Obsidian Plugin accessing jira and displaying its issues.

https://github.com/marc0l92/obsidian-jira-issue

**Comment by max-mykhailenko** [comment]
The main problem for me is that when I put an Outline link into Jira, I don't have a preview because it's a private document. How can we authorize Jira? I can help with plugin development using Jira Forge if some tokens need to be stored on the Jira side. @tommoor, what do you think about this? How can we better compete with Confluence? Do you have such goals?

<img width="768" alt="image" src="https://github.com/user-attachments/assets/1bb9d4c3-7939-4f5a-9eb1-de4a79bcad80">

**Reply by tommoor** [reply]
It would be great to enable rich display in JIRA, I don't have any experience building JIRA plugins to do so but this should be completely possible with the API

**Comment by axelsimon** [comment]
As much as I'm not a fan of Jira, many companies use it (mine included) and being able to refer to Jira tasks and issues, embed them or see previews, etc. could be very useful. So another vote for this.

**Comment by baditaflorin** [comment]
Would also need this, i need to understand more of how the outline system works so I could try something
